provisioning/buffer/aws.sh

```diff
@@ -15,7 +15,19 @@ fi
 # Get credentials to the Amazon Elastic Kubernetes Service
 aws eks update-kubeconfig --name "$AWS_EKS_CLUSTER_NAME" --region "$AWS_REGION"
 
+if [ "${TMP_ETCD_REDEPLOY:-false}" = "true" ]
```

Comment:
Da sa to otestovat aj v CI, ak nastavis env `TMP_ETCD_REDEPLOY` tu (a rovnako pre templates api v inom subore):

https://github.com/keboola/keboola-as-code/blob/23a34f89ca49a1211865b6fb272dd898b0aaad47/.github/workflows/test-k8s-service-buffer.yml#L13-L25

Reply:
V CI ale nebude fungovat provisioning gp3 disku, takže mi přišlo jednoduší to celé otestovat na aws testingu

Reply:
Ok, to len FYI, ... skor som myslel na otestovanie toho zhodenia/nahodenia.